packages/storybook/src/stories/regressions-grid.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { XGrid, useGridApiRef } from '@material-ui/x-grid';
+
+export default {
+  title: 'regressions/Grid',
```

Comment:
Not sure about this approach.
seems a bit segregated and I prefer to test some stories within their thematic folders. Maybe a prefix on the regressions tested stories would allow us to identify which one are tested?

Reply:
I could do a prefix. A bit harder to find and filter but they would be, as you said, in a thematic folder.

Reply:
I guess that also the value of not using a semantic folder. When you want to add a new regression test case, you don't need to care about it.

Reply:
For instance, here, now I need to ask: where should it be?

Reply:
I'm trying something, we will see

Reply:
![image](https://user-images.githubusercontent.com/936978/110139954-e1839300-7dd3-11eb-9e24-f3f09bb7f353.png)

Somewhere around here...
ie Commodity Edit Cell Snap ?
Suffix might actually be better 🤔